pkg/oidc/verify_test.go

```diff
@@ -0,0 +1,220 @@
+package oidc
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
+	"fmt"
+	"github.com/coreos/go-oidc"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"gopkg.in/square/go-jose.v2"
+)
+
+var _ = Describe("Verify", func() {
+	var (
+		ctx context.Context
+	)
+
+	ctx = context.Background()
```

Comment:
This should be sufficient 🙂 
```suggestion
	ctx := context.Background()
```

Reply:
fixed